packages/airnode-deployer/src/cli/index.ts

```diff
@@ -22,17 +22,21 @@ function drawHeader() {
       "|  _  | | '__| '_ \\ / _ \\ / _` |/ _ \\",
       '| | | | | |  | | | | (_) | (_| |  __/',
       '\\_| |_/_|_|  |_| |_|\\___/ \\__,_|\\___|',
+      '',
       `          Airnode v${getNodeVersion()}`,
       `        Deployer CLI v${packageVersion}`,
+      '',
     ].join('\n')
   );
 }
 
 async function runCommand(command: () => Promise<void>) {
   try {
     await command();
-  } catch (err) {
-    loggerUtils.error((err as Error).message);
+  } catch {
+    // We are not logging the error since all errors should be handled by the specific deployer command. An error here
+    // means the command was not successful and should fail with non zero status code.
+    //
     // eslint-disable-next-line functional/immutable-data
     process.exitCode = 1;
   }
```

Comment:
So, you've removed the logging of the error from `commands.ts` and let the exception go higher (which is here) but here you're ignoring it? Is it logged somewhere? Maybe I'm just blind.

Reply:
For Airnode `deploy` I've added the error logging and for `remove` the error is actually logged here and rethrown https://github.com/api3dao/airnode/blob/master/packages/airnode-deployer/src/infrastructure/index.ts#L315

Reply:
Those only log that the error happened they don't log the error itself. Without that, we actually don't know what really happened, just that something failed. We should definitely either log the error itself or throw it.

Reply:
👍 You're right. We should log it.

Reply:
Actually, deployer package uses a pattern where the error is logged using `logger.fail` which outputs the error nicely in the terminal. Then, the error is rethrown. This means that functions should log the error themselves. We do this for all TF commands (https://github.com/api3dao/airnode/blob/master/packages/airnode-deployer/src/infrastructure/index.ts#L64), but there are some places where this is not the case and error is not handled at all (e.g. checking cloud provider state). 

After some thinking I think logging the error here most of the times just prints what was already printed... However, there might be an unhandled error which would be printed only by this place. If we do not log here, we could risk silent errors which is not worth it.